core/http/src/main/java/org/trellisldp/http/TrellisHttpResource.java

```diff
@@ -191,13 +191,13 @@ public void initialize() {
         if (MISSING_RESOURCE.equals(res) || DELETED_RESOURCE.equals(res)) {
             LOGGER.info("Initializing root container: {}", id);
             return trellis.getResourceService().create(Metadata.builder(id).interactionModel(LDP.BasicContainer)
-                    .build(), dataset);
+                    .hasAcl(true).build(), dataset);
```

Comment:
Why `hasAcl(true)` suddenly?

Reply:
That was an oversight. I could add that to a separate commit, if you prefer. But this initialization step explicitly adds an ACL resource to this root resource, so it will always have an ACL present. Depending on how the persistence backend handles this value, it is more correct to explicitly state `hasAcl(true)` in the builder

Reply:
It's cool, just wasn't sure why it was here, but it's here for good reason.